assembl/static2/js/app/components/common/footer.jsx

```diff
@@ -30,11 +29,17 @@ const Footer = ({
   footerLinks,
   hasLegalNotice,
   hasTermsAndConditions,
-  hasCookiesPolicy,
   hasPrivacyPolicy,
   hasUserGuidelines,
   lang
 }: Props) => {
+  const legalPages = {
+    terms: hasTermsAndConditions,
+    legalNotice: hasLegalNotice,
+    cookiesPolicy: true,
```

Comment:
Why is this one always true ? We have a hasCookiesPolicy boolean in the tabscondition query

Reply:
the cookie page needs to be always here because of the cookie configuration section